lib/ui/views/navigation/navigation_view.dart

```diff
@@ -38,6 +41,10 @@ class _NavigationViewState extends State<NavigationView> {
         iconSize: 28,
         elevation: 5,
         items: <BottomNavigationBarItem>[
+          BottomNavigationBarItem(
+            icon: SvgPicture.asset("assets/icons/offers-icon.svg"),
+            label: 'تبديل',
```

Comment:
Make it the third tap between prediction and markets, and use different icon

Reply:
it makes more sense to put it beside p2p trading